app/src/main/java/com/endava/myendava/fragments/ProfileFragment.java

```diff
@@ -187,11 +164,10 @@ private void populateViews(Profile profile) {
         mEmailTextView.setText(profile.getEmail());
         mLocationTextView.setText(profile.getLocation());
         String url = RetrofitClient.TEST_URL + profile.getImageUrl();
-        Log.d("Profile", url);
         Glide.with(getContext()).load(url)
                 .into(mPhotoImageView);
-        adapter = new ChipsAdapter(getContext(), profile,
-                toTagsMap(profile.getTags()), this);
+        adapter = new ChipsAdapter(getContext(), mSharedPreferences.getUserEmail(),false, profile,
```

Comment:
this instance of the adapter is similar to the one above ... why do we need them both?